test/configTestVm.sh

```diff
@@ -1,15 +1,112 @@
 #!/usr/bin/env bash
```

Comment:
I only renamed this file from `doConfigTestVm.sh` to `configTestVm.sh`; nothing in it changed.

This was the original script meant for direct use, then we put a docker wrapper script in front of it, now we're taking the wrapper script back off.